xapian-core/api/query.cc

```diff
@@ -228,6 +228,12 @@ Query::get_subquery(size_t n) const
     return internal->get_subquery(n);
 }
 
+Xapian::termcount
+Query::get_leaf_wqf() const
+{
+	return internal->get_wqf();
```

Comment:
Here (and also in the next file) the code in the function is over-indented - it should be indented by 4 spaces.

Set your editor to display tabs with width 8, to indent by 4 columns, and to tab-fill the indents and you should avoid this problem.